website/templates/public/pages/registries_landing.mako

```diff
@@ -1,19 +0,0 @@
-<%inherit file="base.mako"/>
```

Comment:
This shouldn't be removed.

Keep in mind: "Registries" != "Registered reports". `osf.io/registries` needs to go to the registries ember app, and `/rr/` , etc. need to go to the new onboarder page.

Reply:
Like I mentioned above I don't think this code is ever being hit.

Reply:
@Johnetordoff, this page is just a "nice to have" for local devs. It is hit if you don't have the ember apps setup locally (it would never show up on a prod or staging environment). 

Is it strictly necessary that we keep it? Not really. Does it need to be removed? Not really.